torch/csrc/jit/codegen/cuda/compute_at_map.h

```diff
@@ -56,13 +56,17 @@ class TORCH_CUDA_CU_API ComputeAtMap {
   //   Do not forward through any broadcast IDs
   enum class MappingMode { PARALLEL, LOOP, INDEX };
 
+  // Would be nice to be able to remove this constructor, should be able to do
+  // so if we wrap it in a unique pointer in GPULower
   ComputeAtMap() = default;
-  ComputeAtMap(MappingMode mapping_mode) : mapping_mode_(mapping_mode) {}
-
-  //! Builds all valid mappings. When gpu_lower is not nullptr,
-  //! equivalent mappings for KIR are also created.
-  void build(Fusion* fusion, GpuLower* gpu_lower = nullptr);
 
+  // Passing in a already computed Index map will prevent Loop and Parallel map
+  // construction from having to do it, simply an optimization when available.
+  // Do not pass in an Index map if constructing an Index map.
```

Comment:
Confused with this comment as no index map is passed. Stale comment?

Reply:
Stale, will remove.